src/.vuepress/config.js

```diff
@@ -26,7 +26,7 @@ const sidebar = {
       ]
     },
     {
-      title: 'Components In-Depth',
+      title: 'Componentes en Detalle',
```

Comment:
I think we should use `Componentes en Profundidad` to keep it consistent with the current docs
```suggestion
      title: 'Componentes en Profundidad',
```